projects/1.18/assets/simple-storage-network/storagenetwork/lang/zh_cn.json

```
@@ -1,76 +1,99 @@
+  "block.storagenetwork.collector.guide": "使网络开放并充当其它自动化系统或存储方案的仅输入控制器。例如，一个普通漏斗指向网络上的收集接口，它将正常地将物品输入存储网络可用的格子中，类似于$(l:import_kabel)输入线缆$(/l)。不允许提取任何物品。试试接到其它存储模组上！",
   "block.storagenetwork.collector.tooltip": "将物品输入网络",
   "item.storagenetwork.speed_upgrade": "速度升级",
+  "item.storagenetwork.speed_upgrade.guide": "这个升级可放置在线缆中。它将提高输入速度，原本它不是每刻都执行。通过减少执行的刻间隔来提升速度。和缓慢升级共存时，将导致忽略所有缓慢升级",
+  "item.storagenetwork.speed_upgrade.tooltip": "提高输入/输出的速度",
   "item.storagenetwork.operation_upgrade": "操作升级",
+  "item.storagenetwork.operation_upgrade.tooltip": "处理过滤器中的物品数量；可以覆盖其他升级。",
+  "item.storagenetwork.operation_upgrade.guide": "如果放置在使用允许模式的过滤线缆中，它将尝试输入或输出过滤器中设置的准确数量的物品，而忽略堆叠升级和维护升级。例如，一个允许7个石头和3个泥土的过滤器将在每个周期输出这些数量的物品，而不是默认的4个。",
   "item.storagenetwork.stack_upgrade": "堆叠升级",
+  "item.storagenetwork.stack_upgrade.tooltip": "提升堆叠上限",
+  "item.storagenetwork.stack_upgrade.guide": "这个升级可放置在输入和输出线缆中。它将把线缆单次传输的物品数量提高到完整的一组64个，而不是默认的4个。",
+  "item.storagenetwork.single_upgrade": "单体升级",
+  "item.storagenetwork.single_upgrade.tooltip": "将堆叠大小设置为1",
+  "item.storagenetwork.single_upgrade.guide": "此升级可放置在输入或输出线缆中。它将把线缆单次传输的物品数量减少到1个，而不是默认的4个。和堆叠升级同时使用时，将覆盖堆叠升级",
+  "item.storagenetwork.slow_upgrade": "缓慢升级",
+  "item.storagenetwork.slow_upgrade.tooltip": "减缓输入/输出的速度",
+  "item.storagenetwork.slow_upgrade.guide": "这个升级可放置在输入和输出线缆中。通过增加执行的tick间隔减缓速度",
   "item.storagenetwork.stock_upgrade": "维护升级",
+  "item.storagenetwork.stock_upgrade.tooltip": "保持连接的物品栏中物品数量固定",
+  "item.storagenetwork.stock_upgrade.guide": "如果放置在输出线缆中，对于过滤器中的每个物品，当目标物品栏中该物品的数量少于过滤器中的，线缆将持续输出直到达到过滤器中的数量。相反，装有维护升级的输入线缆将从连接的箱子中的基础物品数量开始，减少物品数量直到达到过滤器中的数量。",
   "item.storagenetwork.builder_remote": "网络远程放置器",
   "item.storagenetwork.builder_remote.tooltip": "绑定到网络和方块进行放置",
   "item.storagenetwork.builder_remote.guide": "绑定方式和别的终端一样，但它没有屏幕。它会搜索你点击的方块，并尝试用网络中相匹配的方块进行放置。",
   "item.storagenetwork.builder_remote.blockstate": "未选择方状态；单击选择",
   "item.storagenetwork.inventory_remote": "无线存储终端",
+  "item.storagenetwork.inventory_remote.tooltip": "在网络根目录上使用以进行远程资源清册访问。",
+  "item.storagenetwork.inventory_remote.guide": "完全远程网络访问，不受距离或维度的限制。类似于$(l:inventory)物品存储工作台$(/l)。可以使用绑定的按键打开。",
   "item.storagenetwork.crafting_remote": "无线合成终端",
+  "item.storagenetwork.crafting_remote.tooltip": "在网络根目录上使用以进行远程资源清册访问。",
   "item.storagenetwork.crafting_remote.guide": "完全远程网络访问，不受距离或维度的限制。集成了一个合成台。类似于$(l:request)存储合成工作台$(/l)",
   "item.storagenetwork.picker_remote": "网络远程选取器",
   "item.storagenetwork.picker_remote.tooltip": "点击一个方块将从网络中拉取相匹配的方块",
   "item.storagenetwork.picker_remote.guide": "绑定方式和别的终端一样，但它没有屏幕。对着一个方块点击它。它会请求一些该方块，是一个还是一组取决于你是否蹲下。",
   "item.storagenetwork.collector_remote": "网络远程收集器",
-  "block.storagenetwork.recipe.invalid": "访问无效:",
+  "item.storagenetwork.collector_remote.tooltip": "收集的物品会被转移到网络中",
+  "item.storagenetwork.collector_remote.guide": "绑定方式和别的终端一样。这个远程终端在你的物品栏中任何一格都可工作（包括饰品栏，但不包括末影箱），因此不需要用手拿着或放在快捷栏。当你捡起任何物品，它都会尝试将物品输入到网络中。",

```

| **审查级别** |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思 |

随便写的

原文：
- **Bind this to the network as normal.  This remote works anywhere in your inventory (including curios but not ender chest), no need to hold it in hand or hotbar.  When you pickup any item, this will insert that item into the network if possible**
```suggestion
  "item.storagenetwork.collector_remote.guide": "绑定方式和别的终端一样。这个远程终端在你的物品栏中任何一格都可工作（包括饰品栏，但不包括末影箱），因此不需要用手拿着或放在快捷栏。你捡起物品时，它会尝试将该物品输入到网络中。",
```